app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,72 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import { CommandError } from '../util'
+import { openDesktop } from '../open-desktop'
+import { ICommandModule, mriArgv } from '../load-commands'
+
+interface ICloneArgs extends mriArgv {
+  pr?: number
+  branch?: string
+}
+
+const command: ICommandModule = {
+  command: 'clone <url|slug> [path]',
+  description: 'Clone a repository',
+  args: [
+    {
+      name: 'url|slug',
+      required: true,
+      description: 'The URL to clone, or the GitHub repo slug to clone',
+      type: 'string',
+    },
+    {
+      name: 'path',
```

Comment:
So this means that Desktop will open that path in Finder/Explorer after cloning the repo?